repo/packages/H/hdfs/0/command.json

```diff
@@ -1,5 +1,5 @@
 {
   "pip": [
-    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0"
+    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1"
```

Comment:
Thanks for clarifying, @abhay-agarwal. So the separate repo is indeed necessary, but the way we access it in Universe is not ideal.  We can merge this now and then @abhay-agarwal could add a JIRA to fix this technical debt and work on it within the next 3 weeks. Does that sound like a good plan, @abhay-agarwal and @jsancio?